app/src/Components/DisplayCards/DisplayCards.tsx

```diff
@@ -102,12 +103,25 @@ function DisplayCards ({ search, sortBy, order }: Props) {
           breakClassName={"break-btn"}
         />
       </div>
-      {isLoading && <h1>Loading...</h1>}
+
+      {isLoading && 
+        <div className="pokemon-spinner-container">
+          <div className="pokemon-spinner"></div>
+        </div>
+      }
+      
       {!isLoading && (!cards || cards.length === 0) && <h1>No cards found</h1>}
```

Comment:
I was thinking we could maybe have an image here that would represent no cards being found. Something like this. 
[https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.deviantart.com%2Fvampireselene13%2Fart%2FPikachu-Face-plant-284306977&psig=AOvVaw289yZBZGSsq5aS7UmdBO2Y&ust=1692306179547000&source=images&cd=vfe&opi=89978449&ved=0CA8QjRxqFwoTCMjInveJ4oADFQAAAAAdAAAAABAE](url)

Totally optional though. And it would maybe be good to still have the text just to insure the user would never get confused.